src/plugins/mission/CMakeLists.txt

```diff
@@ -1,7 +1,3 @@
-
-hunter_add_package(jsoncpp)
```

Comment:
This are unneeded in mission?

Reply:
Json import was moved to mission_raw, so not needed here anymore.